modules/web-console/frontend/app/components/bs-select-menu/strip.filter.js

```diff
@@ -17,6 +17,9 @@
 
 export default function() {
     return function(val) {
-        return val ? val.replace(/(<\/?\w+>)/igm, '') : '';
+        if (typeof val !== 'string')
```

Comment:
Am I correct to assume an additional check is required in case you want to pass SCE'd content in?